### DISCUSSION TITLE
Failed educational discount request and the feedbacks are not accurate/wrong

### DISCUSSION THREAD

**Discussion by HaiyangJin** [community/community#56666] [GitHub Education]
### Hi everyone,

Hi

Recently I educational discount request failed and the suggestions are:

1. The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

2. The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

For the first feedback: The uploaded ID clearly shows "NYU" at the top, so I guess "NYU" does not match "New York University", which is automatically identified from my email address.

For the second feedback: I'm at the Abu Dhabi campus of NYU (https://nyuad.nyu.edu/en/), but the message said "The school you selected does not appear to have a campus location in your country".

Could you please let me know if there is any futher suggestion? Thanks.

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
